ESP8266Interface.h

```diff
@@ -161,6 +161,12 @@ class ESP8266Interface : public NetworkStack, public WiFiInterface
      */
     using NetworkInterface::add_dns_server;
 
+    /** Check if connected to the network
+     *
+     *  @return         true if connected, false otherwise
+     */
+    bool isConnected(void);
```

Comment:
I wouldn't add this function unless it is also added to the network interface, otherwise we may run into name conflicts when we do